calico/felix/dispatch.py

```diff
@@ -164,16 +165,36 @@ def _calculate_update(self, ifaces):
         # Special case: allow the metadata IP through from all interfaces.
         if self.config.METADATA_IP is not None and self.ip_version == 4:
             # Need to allow outgoing Metadata requests.
-            root_from_upds.append("--append %s "
-                                  "--protocol tcp "
-                                  "--in-interface %s+ "
-                                  "--destination %s "
-                                  "--dport %s "
-                                  "--jump RETURN" %
-                                  (CHAIN_FROM_ENDPOINT,
-                                   self.config.IFACE_PREFIX,
-                                   self.config.METADATA_IP,
-                                   self.config.METADATA_PORT))
+            root_inbound_upds.append("--append %s "
```

Comment:
Sorry, didn't get to this yesterday; is there any reason this chain needs to be managed by the dispatch chain actor, seems like it's a static chain so we could even write the contents directly into felix-INPUT?

The reason might be that you can see things that we'll want to put in here later that depend on the list of tap devices.